turbo/mixins.py

```diff
@@ -14,40 +14,69 @@
 
 
 class BroadcastableMixin(object):
+    default_stream_action = APPEND
     broadcasts_to = []  # Foreign Key fieldnames to broadcast updates for.
-    broadcast_self = True  # Whether or not to broadcast updates on this model's own stream.
-    inserts_by = APPEND  # Whether to append or prepend when adding to a list (broadcasting to a foreign key).
     turbo_streams_template = None
 
-    def get_turbo_streams_template(self):
-        if self.turbo_streams_template is not None:
-            return self.turbo_streams_template
-        app_name, model_name = self._meta.label.lower().split(".")
-        return f"{app_name}/{model_name}.html"
+    def get_turbo_streams_template(self, target):
+        raise Exception('No Template known!')
```

Comment:
I beleive this should be a `NotImplementedError`

Reply:
This is a good question and indeed it makes use of the model as I re-use the same templates and thus need a model. I built the second example with the "broadcast" view as another example (but it again uses the same template and thus the model).
Would it help to built another demo which does not at all rely on Models but just does some "messaging"?

Generally, I agree with you in that I'm unsure if thats the right abstraction or there even is a need for such a Mixin (where should it be mixed in?). Thus I added the "/broadcast" view in another commit.

At least I tried to disentangle the "model based" logic from the "stream based" logic.

What would you say, would be good next steps to evaluate this question? Replicate the whole Chat-Demo but Model-free? Or should we talk about a totally different use case?